src/peripherals/usart.c

```diff
@@ -183,9 +184,9 @@ void usarts_enable(u32 ftdi_baud, u32 uarta_baud, u32 uartb_baud, bool do_precon
 
   if (do_preconfigure_hooks) {
 
-    printf("\n\nPiksi Starting...\n"
-       "Firmware Version: " GIT_VERSION "\n" \
-       "Built: " __DATE__ " " __TIME__ "\n");
+    log_info("Piksi Starting...\n");
```

Comment:
I kind of like the idea of this being a log_warn.  It's nice to have a message on startup from a usability perspective and with this change I don't think anyone will see the Piksi starting and the firmware version.